index.html

```diff
@@ -661,6 +668,33 @@
       </section>
       <section>
         <h2>
+          Non-constrainable properties
+        </h2>
+        <p>
+          A <dfn lt=
+          "non-constrainable property|non-constrainable properties">non-constrainable
+          property</dfn> is ignored in the context of the
+          <a>applyConstraints()</a> algorithm. Its value can be obtained via
+          the <a>getSettings()</a> accessor. A <a>non-constrainable
+          property</a>'s <a>fitness distance</a> is always 0.
```

Comment:
This new thing is not in the constrainable algorithm today, and seems unnecessary. I see nothing harmful in letting people putting constraints on fixed values. Worst case `applyConstraints` just fails.